CHANGELOG.latest.md

```diff
@@ -1,37 +1,6 @@
-The first beta of **RevenueCat Purchases React Native v6** is here!! 😻
+### New Features
+* Add offering metadata (#641) via Josh Holtz (@joshdholtz)
+### Other Changes
+* Bumped fastlane internal plugin via Josh Holtz (@joshdholtz)
+* Fix issue where long values weren't getting sent and fix local testing (#636)
```

Comment:
Should we also include all the changelog from 6.0.0-beta.1?

Reply:
Good call good call!